### ISSUE TITLE
DAPS generates ODG->SVG files into images/src (3.0 RC1)

### ISSUE THREAD

**Issue by ghost** [openSUSE/daps#476]
Building `daps -d DC-SLES-security -vv --force html` from doc-sle with current DAPS has DAPS regenerate SVG files (which is fine, I guess). However, it generates them into `images/src/` and not into `images/gen(erated)/` or `build/` or whatever it used to be.

I don't think this is what we want -- usually everything in `images/src` is committed to git and I don't think normal `daps` subcommands[1] should create anything into paths under which files will usually be committed to git. In addition, those SVGs actually existed before, so were overwritten.

[1] exceptions are of course `daps optipng` and `daps xmlformat`

**Comment by fsundermeyer** [comment]
I cannot reproduce this behaviour. Can you please double-check and provide log files (please run daps --debug)?

**Comment by ghost** [comment]
It's a bit weird. It only seems to happen when I run the command twice, so make that:

```
daps -d DC-SLES-security -vv --force html
daps -d DC-SLES-security -vv --force html
```

**Comment by fsundermeyer** [comment]
Fixed with 9f7456d
